diff --git a/core/user_tags.mk b/core/user_tags.mk
index 2e7017e..de85594 100644
--- a/core/user_tags.mk
+++ b/core/user_tags.mk
@@ -72,6 +72,7 @@ GRANDFATHERED_USER_MODULES += \
 	com.android.phone.common \
 	com.android.vcard \
 	commons-compress-1.0 \
+	copybit.msm7k \
 	copybit.qsd8k \
 	copybit.s5pc110 \
 	coverage \
@@ -133,6 +134,7 @@ GRANDFATHERED_USER_MODULES += \
 	genext2fs \
 	gps.mahimahi \
 	gralloc.default \
+	gralloc.msm7k \
 	gralloc.qsd8k \
 	groovy-all-1.7.0 \
 	grxmlcompile \
@@ -379,6 +381,7 @@ GRANDFATHERED_USER_MODULES += \
 	libxslt \
 	libzipfile \
 	lights.kraken \
+	lights.msm7k \
 	lights.qsd8k \
 	line_endings \
 	linker \
@@ -490,6 +493,7 @@ GRANDFATHERED_USER_MODULES += \
 	test-progress-new \
 	test_swiarb \
 	test_zipfile \
+	tiwlan.ini \
 	toolbox \
 	traceview \
 	tune2fs \
@@ -499,7 +503,9 @@ GRANDFATHERED_USER_MODULES += \
 	vdc \
 	vm-tests \
 	vold \
+	wlan_loader \
 	wdsclient \
 	wpa_supplicant \
+	wpa_supplicant.conf \
 	yuv420sp2rgb \
 	zipalign
